### DISCUSSION TITLE
Feature Request: Adding Prefix Option for DaisyUI Class Names when Imported via CDN

### DISCUSSION THREAD

**Discussion by kknono** [saadeghi/daisyui#2431] [Ideas / feature requests]
Tailwind and DaisyUI are both imported through CDN. How can a prefix be provided for DaisyUI? For example, changing` btn `to `dy-btn.`

please help me.

**Comment by saadeghi** [comment]
It's not possible. The CDN file is a static CSS file.  

But may I ask why are you using the CDN file?
